test/functional/ex_cmds/make_spec.lua

```diff
@@ -2,25 +2,30 @@ local helpers = require('test.functional.helpers')(after_each)
 local clear = helpers.clear
 local eq = helpers.eq
 local eval = helpers.eval
+local has_powershell = helpers.has_powershell
 local nvim = helpers.nvim
 local nvim_dir = helpers.nvim_dir
 
 describe(':make', function()
 
   describe('with powershell', function()
+      before_each(function ()
+        clear()
+        if has_powershell() then
```

Comment:
Can you skip the entire test suite instead if the user doesn't have powershell?

Reply:
Sounds good, i will try that. I was going by the README instruction which states
```
skip the test via pending() if the external factor is missing. This ensures that the total test-count (success + fail + error + pending) is the same in all environments.
```

Reply:
Done; PTAL